### ISSUE TITLE
Assignment-4 : Reverse String

### ISSUE THREAD

**Issue by 41bit23** [vtcbca/basic-programming-62bit23#5]
Can a string with special characters be reversed?

**Comment by 62bit23** [comment]
Yes, special characters are treated like regular characters and will appear in reverse order.

**Comment by 41bit23** [comment]
What happens when a string is reversed twice?

**Comment by 62bit23** [comment]
Reversing a string twice returns the original string

**Comment by 41bit23** [comment]
How can you reverse a string without using slicing or built-in functions?

**Comment by 62bit23** [comment]
Use a loop to iterate through the string from the end to the beginning and concatenate the characters

**Comment by 41bit23** [comment]
Can strings in Python be reversed in place?

**Comment by 62bit23** [comment]
No, strings in Python are immutable, so you need to create a new string for the reversed version
